services/harmony/test/collection-capabilities.ts

```diff
@@ -74,7 +74,7 @@ describe('Testing collection capabilities', function () {
         it('sets the outputFormats field correctly', function () {
           const capabilities = JSON.parse(this.res.text);
           const expectedFormats = [
-            'application/x-netcdf4', 'image/tiff', 'image/png', 'image/gif', 'application/x-zarr',
+            'application/x-netcdf4', 'image/tiff', 'application/x-zarr', 'image/png', 'image/gif',
```

Comment:
Just a quick explanation on these updated tests:

Harmony concatenates all the output formats into a single list. The ordering of the list reflects the first time each output is encountered in `services.yml`. HGA is listed above NetCDF-to-Zarr, which in turn is above harmony-service-example.

Before this change, HGA and harmony-service-example both listed PNGs and GIFs as output options, so those formats were listed before Zarr, because HGA is before NetCDF-to-Zarr. After this change, only harmony-service-example lists PNG and GIF, so these formats are found after NetCDF-to-Zarr and listed after the Zarr output option.

Kind of in-the-weeds, but wanted to explain what looks like a weird test change.